CODEOWNERS

```
@@ -0,0 +1,27 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @Rocketknight1 # if no one is pinged based on the other rules, he will do the dispatch
+**.md @stevhliu
+docs/ @stevhliu
+/benchmark/ @McPatate
+/src/transformers/models/*/*processing* @molbap @yonigozlan @qubvel
+/src/transformers/models/*/image_processing* @qubvel
+/src/transformers/models/*/image_processing_*_fast* @yonigozlan
+# /src/transformers/models/*/*_modeling* depending on the type of the model, @qubvel @NielsRogge @Cyrilvallez @zucchini-nlp @molbap 
+/src/transformers/models/*/*_modeling* @Rocketknight1 # for appropriate dispatch !a
```

```suggestion
/src/transformers/models/*/*_modeling* @arthurzucker
```
will just go with this for now, I don't think it makes sense to assign individual models, but modalities to people

---

Oh, you can assign me for dispatch, in that case! You don't need to take the extra work